backend/cms/templates/pages/page.html

```diff
@@ -0,0 +1,168 @@
+                <div class="mt-4">
+                    <!--{% render_field form.text placeholder="Text" class="tinymce_textarea" %}-->
+                    {% render_field form.text placeholder="Text" class="bg-grey-lighter w-full p-2" %}
+                </div>
+            </div>
+            <div class="w-full p-4 rounded border border-solid border-grey-light shadow bg-white">
+                <h3 class="mb-2">Live Inhalte einbinden</h3>
+                <input type="text" class="appearance-none block w-full bg-grey-lighter text-grey-darker border border-grey-lighter rounded mb-2 py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-grey"
+                    placeholder="Seite suchen" />
+                <div class="relative my-2">
+                    <select class="block appearance-none w-full bg-grey-lighter border border-grey-lighter text-grey-darkest py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-grey">
+                        <option>Davor anhängen</option>
+                        <option>Danach anhängen</option>
+                    </select>
+                    <div class="pointer-events-none absolute pin-y pin-r flex items-center px-2 text-grey-darkest">
+                        <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+                            <path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" /></svg>
+                    </div>
+                </div>
+                <button class="bg-blue hover:bg-blue-dark text-white font-bold py-2 px-4 rounded">
+                    Seite anhängen
+                </button>
+            </div>
+        </div>
+        <div class="w-1/3 pl-2">
+            <div class="w-full px-4 py-2 rounded border border-solid border-grey-light shadow bg-white">
+                <div class="py-2 border-b solid border-grey-lighter mb-2">
+                    <span class="block mb-2 font-bold">Sprache</span>
+                    <label class="text-xs uppercase block">Aktuelle Sprache</label>
+                    <div class="relative my-2">
+                        {% render_field form.language id="parent" class="block appearance-none w-full bg-grey-lighter border border-grey-lighter text-grey-darkest py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-grey" %}
+                        <div class="pointer-events-none absolute pin-y pin-r flex items-center px-2 text-grey-darkest">
+                            <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+                                <path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
+                            </svg>
```

Comment:
Can we move all these svgs to separate files in some resources folder? with an appropriate name